scripts/run-playwright-container.sh

```diff
@@ -4,4 +4,4 @@ set -e
 CONTAINER_RUNNER="${CONTAINER_RUNNER:-podman}"
 PLAYWRIGHT_VERSION=$(scripts/get-playwright-version.sh)
 
-$CONTAINER_RUNNER run --rm -v "$(pwd)":/work/ -w /work/ -it -e CI mcr.microsoft.com/playwright:"${PLAYWRIGHT_VERSION}" "$@"
+$CONTAINER_RUNNER run --rm -v "$(pwd)":/work/ -v /work/node_modules -w /work/ -it -e CI mcr.microsoft.com/playwright:"${PLAYWRIGHT_VERSION}" "$@"
```

Comment:
oh this was not meant to be committed. 
I will make a fix up to remove this.  
This is how I do the workaround for running  playwright. 
I had to mount a volume to node_modules because I was getting permission errors when running npm install

Reply:
It might be needed to be honest because the packages could be different between macOS and Ubuntu.